homeassistant/components/touchline/climate.py

```diff
@@ -61,6 +61,7 @@ class Touchline(ClimateEntity):
 
     _attr_hvac_mode = HVACMode.HEAT
     _attr_hvac_modes = [HVACMode.HEAT]
+    _attr_should_poll = False
```

Comment:
The property returned True.

Reply:
```suggestion
```

Reply:
Indeed! I thought I had excluded those beforehand.